pkg/management/postgres/metrics/parser.go

```diff
@@ -29,6 +29,7 @@ type UserQueries map[string]UserQuery
 // UserQuery represent a query created by the user
 type UserQuery struct {
 	Query           string    `yaml:"query"`
+	PredicateQuery  string    `yaml:"predicate_query"`
```

Comment:
Perhaps a more explicit name would be better?
writing this to gather ideas

Reply:
To me "predicate query" is the right term - it's a query that is used as a predicate to determine whether the main query should run.

Suggestions are welcome. `ConditonalExecutionQuery` ? IDK.

Reply:
`predicate_query` is good to me however feel free to propose the name that you think fits better the cnp config 😄

Reply:
@armru Would you be happier with `ConditionQuery` or something? Suggestions welcome if you don't like the current naming.

Reply:
We can leave `PredicateQuery` then :) I was opening a discussion to gather ideas